packages/jest-expo/README.md

```diff
@@ -30,6 +30,22 @@ at https://github.com/expo/expo. Thanks!
 
 - Run `npm test` and it should pass
 
+## Platforms
+
+You can use `jest-expo` to test any Expo supported platform. For legacy purposes `jest-expo` runs your tests in the standard React Native environment (iOS).
+The recommended way to test your project is with `jest-expo/universal` which runs your tests with every Expo supported platform. Currently this includes iOS, Android, web, and Node (which is used for testing SSR compliance).
+
+For quick debugging you can run a single platform with `jest-expo/ios`, `jest-expo/android`, `jest-expo/web`, and `jest-expo/node`.
```

Comment:
since it's implemented with `projects`, maybe include a link to https://www.npmjs.com/package/jest-watch-select-projects? A watch plugin allowing you to select a project rather than quitting jest, changing the config, then re-running it

---

I absolutely love what you've done here btw, so cool! 👏

Reply:
Hey sorry I missed this. Great suggestion thanks for the link :D